looppointer.go

```diff
@@ -176,3 +170,15 @@ func getIdentity(expr ast.Expr) *ast.Ident {
 	}
 	return nil
 }
+
+func selectorRoot(selector *ast.SelectorExpr) *ast.Ident {
```

Comment:
:thought_balloon: Maybe we can write them more simple

Reply:
Almost certainly. 😅  I'm looking forward to see how you edit it.